tests/layer_tests/pytorch_tests/test_log_softmax.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2018-2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import pytest
+import torch
+import torch.nn.functional as F
+
+from pytorch_layer_test_class import PytorchLayerTest
+
+class aten_log_softmax(torch.nn.Module):
+    def __init__(self, dim, dtype) -> None:
+        super().__init__()
+        self.dim = dim
+        self.dtype = dtype
+
+    def forward(self, input_tensor):
+        return F.log_softmax(input_tensor, dim = self.dim, dtype = self.dtype)
+
+class TestLogSoftmax(PytorchLayerTest):
+    def _prepare_input(self):
+        return (self.input_tensor,)
+
+    @pytest.mark.parametrize(["input_tensor", "dtype"], [
```

Comment:
Please reimplement to create tensor in `_prepare_input` otherwise it makes test name too long and makes it much harder to debug such test fails

Reply:
Done!